FuelLabs/fuels-portal
# fix: bugfix + e2e tests improvements

Closes #163 
Closes FRO-651

## Bugfix
Bug when clicking to withdraw under this situation:
- Select withdraw Tab > Click in Transaction Tab > Click in Bridge Tab. 
- Withdraw tab will be selected, but with deposit data in the form.


## Improve E2E testing

### Refact / subgroups
As tests are sometimes flaky and long, it's hard to debug where is the problem
To solve this:
- Splitted more the tests into subgroups
- Described better each steps
- Isolated wallet approvals

### Increase scenarios coverage
UX experience:
- test loading state in deposit dialog
- skip test loading state in withdraw dialog (FUEL blazingly fast)
- check if TX is settled on list after transaction
- check assets in selector over the process
- check time left shown in withdraw popups
- check TX list after refreshing, all transactions made should be "Settled"
- check if bridge page after refreshing, is still connected to same wallet and has same balance
